File: src/main/java/dev/sky_lock/pocketlifevehicle/task/BurnExplosionWarning.java
package dev.sky_lock.pocketlifevehicle.task;

import org.bukkit.ChatColor;
import org.bukkit.entity.Player;

/**
 * @author sky_lock
 */

class BurnExplosionWarning extends AbstractWarning {
    void send(Player player) {
        super.send(player, ChatColor.GOLD + "乗り物は" + getCount() + "秒後に爆発します");
    }
}

File: src/main/kotlin/dev/sky_lock/pocketlifevehicle/task/BurnExplosionWarning.kt
package dev.sky_lock.pocketlifevehicle.task

import dev.sky_lock.pocketlifevehicle.text.Line
import org.bukkit.entity.Player

/**
 * @author sky_lock
 */
class BurnExplosionWarning : AbstractWarning() {
    fun send(player: Player) {
        super.send(player, Line().gold("乗り物は" + count + "秒後に爆発します"))
    }
}